fapi-client/src/main/scala/com/gu/facia/api/utils/ItemKicker.scala

```diff
@@ -85,7 +85,7 @@ object ItemKicker {
 
   def seriesOrBlogKicker(item: Content) =
     item.tags.find({ tag =>
-      Set("series", "blog").contains(tag.`type`) && !TagsThatDoNotAutoKicker.contains(tag.id)
+      Set(TagType.Series: TagType, TagType.Blog).contains(tag.`type`) && !TagsThatDoNotAutoKicker.contains(tag.id)
```

Comment:
That is effectively what annotating the type of the first element is doing, but better.  Thanks, I will change it